tests/app.test.js

```
@@ -3,171 +1,172 @@
+const express = require('express');
 const {
-    NekoGirlStateMachine,
-    Emotion,
-    Animation,
-} = require("../NekoGirlStateMachine");
-const { EventEmitter } = require("events");
+  NekoGirlStateMachine,
+  Emotion,
+  Animation,
+} = require('../NekoGirlStateMachine');
+const { EventEmitter } = require('events');
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)